eng/common/scripts/Create-APIReview.ps1

```diff
@@ -41,11 +41,12 @@ function Submit-APIReview($packagename, $filePath, $uri, $apiKey, $apiLabel)
     try
     {
         $Response = Invoke-WebRequest -Method 'POST' -Uri $uri -Body $multipartContent -Headers $headers
-        Write-Host "API Review: $($Response)"
+        Write-Host "API Review URL: $($Response)"
         $StatusCode = $Response.StatusCode
     }
     catch
     {
+        Write-Error "Exception details: $($_.Exception.Response)"
```

Comment:
Should we exit 1 here?

Reply:
I see this is a reusable function. However given you are calling Write-Error in some contexts like in DevOps that will cause the script to stop.

Reply:
I have changed it to Write-Host and let it fall through calling code.